src/components/Layout/BurgerMenu.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useContext } from 'react'
+import { BurgerMenuShowContext } from '../Context/BurgerMenuShowContext'
+import { BurgerMenuStatusContext } from '../Context/BurgerMenuStatusContext'
+import './BurgerMenu.css'
+
+function BurgerMenu() {
+  const statusContext = useContext(BurgerMenuStatusContext)
+  const showContext = useContext(BurgerMenuShowContext)
+
+  const toggleMenuHandler = () => {
+    statusContext?.setStatus(prev => !prev)
+  }
+
+  return <div onClick={toggleMenuHandler} style={showContext?.show ? {} : {display: 'none'}}>
```

Comment:
Maybe it is cleaner if we can toggle a css class instead of using inline styles. I see the TokenList util being used in other places in the code.

```
<div style={TokenList.join([!showcontext?.show && 'BurgerMenu--hidden'])} />
```